### PR TITLE
Update ietf-bearer-svc.yang

### PR DESCRIPTION
**Pull request by Ivan-Bykov-RBBN**

### CODE REVIEW

File: yang/ietf-bearer-svc.yang [hunk 1]
```diff
@@ -111,6 +111,11 @@ module ietf-bearer-svc {
   grouping location-information {
     description
       "Basic location information";
+    leaf locationName {
+        type string;
+        description
+          "Refers to 3GPP NRM {{TS-28.623}} Clause A 2.2.2 IOC ManagedElement. The physical location (e.g., an address) of an (3GPP) entity. It may contain no information to support the case where the derivative of ManagedElement needs to represent a distributed multi-location NE.";
```
Comment (boucadair) [inline yang/ietf-bearer-svc.yang:117]:
```suggestion
          "Provides a location name. This data node can be mapped, e.g., to the 3GPP
           NRM IOC ManagedElement.";
```

Comment (boucadair) [reply yang/ietf-bearer-svc.yang:117]: Thank @Ivan-Bykov-RBBN. The description should be generic. It is OK to provide NRM as an example.
Please let me know if the proposed change is OK with you.

Comment (Ivan-Bykov-RBBN) [reply yang/ietf-bearer-svc.yang:117]: Agreed.
Thank you!

File: yang/ietf-bearer-svc.yang [hunk 2]
```diff
@@ -111,6 +111,12 @@ module ietf-bearer-svc {
   grouping location-information {
     description
       "Basic location information";
+    leaf locationName {
```
Comment (boucadair) [inline yang/ietf-bearer-svc.yang:114]:
```suggestion
    leaf location-name {
```

**Comment by boucadair** [comment]
Updated the PR as per our agreement in the call: move the location-name to be under location. The same data node will also be used for the site as well.

**Pull request merged**